miio/walkingpad.py

```diff
@@ -41,7 +41,7 @@ class WalkingpadStatus(DeviceStatus):
      'sp': 3.0,
      'start_speed': 3.0,
      'step': 180,
-     'time': 121}
+     'walking_time': 121}
```

Comment:
The code below uses `self.data["time"]` for `walking_time`. Which is the correct input?

Reply:
good catch, you're right it should stay as time. I have corrected.